dragodis/ida/disassembler.py

```diff
@@ -506,6 +514,15 @@ def _async(self, proxy_func) -> Callable[_, rpyc.AsyncResult]:
         Good for functions that we don't care to get results back from.
         """
         return rpyc.async_(proxy_func)
+    
+    def analyze(self) -> None:
+        """
+        Instruct IDA to initiate and wait for auto analysis completion.
+        """
+        logger.debug('Running autoanalysis.')
```

Comment:
Can we add a check to ensure the `_running` flag is enabled before executing? This will ensure the user doesn't run this function without starting the disassembler first.